website/common/script/libs/getItemInfo.js

```diff
@@ -237,7 +237,7 @@ module.exports = function getItemInfo (user, type, item, officialPinnedItems, la
         notes: item.notes(language),
         value: item.value,
         currency: 'gold',
-        purchaseType: 'potions',
```

Comment:
Will this string change affect the client checks at all? I think these hard coded strings are used elsewhere

Reply:
it was changed to potion because `purchaseType` is assigned to `type` in 

```
makeGenericPurchase (item, type = 'buyModal', quantity = 1) {
      this.$store.dispatch('shops:genericPurchase', {
        pinType: item.pinType,
        type: item.purchaseType,
        key: item.key,
        currency: item.currency,
        quantity,
      });
```

then `type` is used for the op-method in `buy.js`